Jenkinsfile.uploadcliqz

```diff
@@ -42,7 +42,7 @@ node('master') {
                             cd mozilla-release
                             export MOZ_CHROME_MULTILOCALE=`ls -1 "../l10n/"|paste -s -d " "`
                             export AB_CD=multi
-                            ./gradlew assembleOfficialWithGeckoBinariesNoMinApiPhotonRelease
+                            ./gradlew :app:assembleWithGeckoBinariesRelease
```

Comment:
Line No 48, 51, 88, 91 of `Jenkinsfile.uploadcliqz` needs to be changed as well. To search/upload the APK from the right folder.